src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -29,6 +29,9 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
   return (
     <div className="schedule-item-container">
       <div className={`time-section ${isCurrentEvent ? "primary" : ""}`}>
+        <div>
+          <b>{labelDayRoomSchedule(event.start_utc_seconds)}</b>
+        </div>
```

Comment:
What's the intent of this change?

Reply:
We don't want to use `<b>` tags in general. Either use `<strong>`, or even better, following a similar pattern as used in `src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.tsx` with the `<span>` and a semanticly BEM named class.

Reply:
The intent of the change is that currently all events in the room modal are only showing the time irrespective of the day. If the event is now occuring today, this will display 'tomorrow' in the case of tomorrow, and `[Abbrev Month] Date` if the event is occuring after tomorrow.

Reply:
@sparkletown/engineering make sure we find and link the backlog issue about this from the internal board to this PR with closing keywords

Reply:
This is the related issue https://github.com/sparkletown/internal-sparkle-issues/issues/783